ui/perfherder/alerts/StatusDropdown.jsx

```diff
@@ -44,9 +45,79 @@ export default class StatusDropdown extends React.Component {
       showNotesModal: false,
       showTagsModal: false,
       selectedValue: this.props.issueTrackers[0].text,
+      jobList: [],
     };
   }
 
+  containsVismet = (title) => {
+    const contains = visualMetrics.find((metric) => title.includes(metric));
+    return contains;
+  };
+
+  anyAlertWithVideoResults = (alertSummary) => {
+    // For the moment the browsertime vismet are separate from the pageload ones
+    // that's why we need to filter them out. Also, for the moment we're retrieving
+    // the video results for regressions only
+    const any = alertSummary.alerts.filter(
+      (alert) =>
+        !this.containsVismet(alert.title) && alert.is_regression === true,
+    );
+    return any.length > 0;
+  };
+
+  isRelevantAlert = (alert) => {
```

Comment:
The term "relevant" is very generic & it's even used on the backend side with a different meaning.
We should instead give this method a more specific name. Something to hint that the alert has videos or not.

Reply:
please suggest, I'm out of ideas.

Reply:
What about `shoulHaveVideoLinks()`?